stock_voucher/security/stock_voucher_security.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="stock_book_rule" model="ir.rule">
+        <field name="name">Stock Book Multi-Company</field>
+        <field name="model_id" ref="model_stock_book"/>
+        <field eval="True" name="global"/>
```

Comment:
review if this one is really needed, As far as I know if you do not add any group then this option is automatically set to True

+        <field eval="True" name="global"/>